android/PhysicalWeb/app/src/main/java/org/physical_web/physicalweb/NearbyBeaconsFragment.java

```diff
@@ -466,6 +469,10 @@ public void clear() {
       mUrlToDeviceAddress.clear();
       notifyDataSetChanged();
     }
+
+    public void redrawListView() {
```

Comment:
Good question. Mainly for clarity. While really calling notifyDataSetChanged, the reason for it is to redraw the listView. Also, since notifyDataSetChanged is overridden, I can't call the original notifyDataSetChanged (outside the class) without making a new method with a different name.